manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -19,7 +19,6 @@ spec:
         image: quay.io/openshift/origin-cluster-network-operator:4.3
         command:
         - "/usr/bin/cluster-network-operator"
-        - "--url-only-kubeconfig=/etc/kubernetes/kubeconfig"
```

Comment:
in KIND we don't need it, the controller is able to get the config from the pod environment.
Should I add a new deployment yaml and keep this?